README.adoc

```diff
@@ -136,27 +138,29 @@ configuration elements. Use these credentials to view the metrics endpoint.
 
 === Adding the annotations
 
-Create the `src/main/java/io/openliberty/guides/inventory/InventoryManager.java` class:
+Create the `InventoryManager` class in the 
+`src/main/java/io/openliberty/guides/inventory/InventoryManager.java` file::
 
 [source, java, indent=0]
 ----
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[tags=InventoryManager]
 ----
 
-You added the following metrics annotations:
+Notice these metrics annotations:
 
 The `@Timed` annotation applies to the `get()` method to track how frequently the method is
-invoked and also how long it takes for the invocation to complete. This annotation is
-followed by fields that correspond to metadata fields:
+invoked and also how long it takes for the invocation to complete. This annotation has these 
+metadata fields:
 
 |===
 |`name` | Optional and sets the name of the metric
 |`description` | Optional and describes the purpose of the metric annotation
 |===
 
-The `@Counted` annotation applies to the `list()` method. This annotation counts how many times
-you request the list of systems methods by visiting the `http://localhost:9080/inventory/systems` URL to
-retrieve the information for a list of all previously registered hosts. Below are the explanation of the fields:
+The `@Counted` annotation applies to the `list()` method. This annotation counts how many times 
+you request the list of systems methods when you visit the `http://localhost:9080/inventory/systems`
+URL that retrieves the information for a list of all previously registered hosts. These are the 
+additional metadata fields: 
```

Comment:
The `@Counted` annotation applies to the `list()` method. This annotation counts how many times you request the list of systems methods when you visit the `http://localhost:9080/inventory/systems` URL. This URL retrieves the information for a list of all previously registered hosts. The following table shows the additional metadata fields:

Reply:
"methods" should be singular.

Reply:
I like "Here are the additional metadata fields:" if that's ok. @sishida

Reply:
Fixed

Reply:
=> "See the additional metadata fields:"

Reply:
Fixed